x-pack/plugins/observability_solution/observability/public/components/custom_threshold/helpers/get_search_configuration.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { defaultQuery, getSearchConfiguration } from './get_search_configuration';
+
+describe('getSearchConfiguration()', () => {
+  const onWarning = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
@maryam-saeidi I don't see any mocks in this file. Maybe this is redundant?

Reply:
I have a `onWarning = jest.fn()` above which is technically a mock, isn't it?

Reply:
Yep definitely. I somehow missed it!